src/tools/decomp.py

```diff
@@ -245,6 +355,7 @@ def nodeGridSimple(n):
                     d2 = j
                     d3 = i
                     ijkmax = i * i + j * j + k * k
+    print d1, d2, d3	
```

Comment:
Is this an extra arbitrary print?

Reply:
yes, I added it to know that HeSpaDDA is not used in this cases. Morevoer, I should better explicitly write that when the nodeGridSImple(n) is used HeSpaDDA is not doing anything. Thanks!

Reply:
Add some prefix string, I really hate it when 3 numbers randomly show up on my screen without an idea from where.

Reply:
done!